web/source/keyboards/kmwkeyboards.ts

```diff
@@ -847,6 +856,9 @@ namespace com.keyman {
     getKeyboardModifierBitmask(k0?) {
```

Comment:
This function has a different pattern for `k`, `k0` and `textProcessor.activeKeyboard` than say, `getFont`. Not sure which is correct usage.

Second question: does `k0` parameter point to the `scriptObject` directly?

Reply:
It does.  This is reworked in the child PR.